apps/channel/client/index.coffee

```diff
@@ -28,12 +29,14 @@ module.exports = ->
     mode: 'skeleton'
 
   initChannelPath(channel)
+
   initShare()
-  initManageCollaborators
-    collection: collaborators
-    current_user: current_user
-  initCollaboratorsList
-    collection: collaborators
+
+  $collaboratorsListEl = $('.js-collaborators-list')
+  mount CollaboratorsList, {
+    channel_id: channel.id
+    htmlFragment: $collaboratorsListEl.html(),
```

Comment:
Ah I see, I think this is a fine compromise for now if you do.

Reply:
Yeah, I'll get to it. The benefit is just being able to delete the old code. We'll be using @artsy/stitch here. Todos, todos.